official-services.md

```diff
@@ -11,10 +10,7 @@ The following services are officially operated by the FEniCS Project.
 - [PyPi](https://pypi.org/user/fenicsproject/)
 - [Slack](https://fenicsproject.slack.com)
 - [Steering Council mailbox](mailto:fenics-steering-council@googlegroups.com)
-- [Support mailing list](https://groups.google.com/forum/#!forum/fenics-support)
-- [Twitter feed curated from #fenicsnews tag](https://twitter.com/search?q=%23fenicsnews&src=typd)
```

Comment:
Unsure if we want to keep this in the list.

Reply:
(this = tweeter)

Reply:
Agree it should be removed as API changes broke it and we have no plans to fix it